SimpleSaver/README.md

```diff
@@ -0,0 +1,56 @@
+# Simple-saver
+
+Dependent on [StreamSaver.js](https://github.com/jimmywarting/StreamSaver.js).
```

Comment:
Add a simple sentence to describe what does this lib do, like

This library helps you to xxxx